modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/registry/DDMServiceUpgradeStepRegistrator.java

```diff
@@ -694,6 +694,11 @@ protected String[][] getTableAndPrimaryKeyColumnNames() {
 				"DDMFormInstance", "expirationDate"));
 
 		registry.register("7.0.3", "7.0.4", new PollsToDDMUpgradeProcess());
+
+		registry.register(
+			"7.0.4", "7.0.5",
+			UpgradeProcessFactory.alterColumnType(
+				"DDMField", "fieldName", "VARCHAR(500) null"));
```

Comment:
@vngoliveira you should also change [here](https://github.com/carolmariaabb/liferay-portal/blob/af5e046ae69035205f8b9d92f9ab1183b6757b85/modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/META-INF/portlet-model-hints.xml#L65) and after that you should run a `buildService`